config/default.yaml

```diff
@@ -0,0 +1,173 @@
+        resources:
+          cpu:
+            # Number of cpu cores
+            micro: "0.5"
+            low: 2
+            high: 6
+            turbo: 12
+          memory:
+            # Memory in GB
+            micro: 1
+            low: 2
+            high: 12
+            turbo: 16
+        # Default build timeout for all builds in this cluster
+        buildTimeout: 90
+        # Default max build timeout
+        maxBuildTimeout: 120
+        # k8s node selectors for approprate pod scheduling
+        nodeSelectors: {}
+        preferredNodeSelectors: {}
+        annotations: {}
+      # Launcher image to use
+      launchImage: screwdrivercd/launcher
+      # Container tags to use
+      launchVersion: stable
+      # Circuit breaker config
+      fusebox:
+        breaker:
+          # in milliseconds
+          timeout: 10000
+      # requestretry configs
+      requestretry:
+        # in milliseconds
+        retryDelay: 3000
+        maxAttempts: 5
+  k8s-vm:
+    options:
+      # Configuration of Docker
+      kubernetes:
+        # The host or IP of the kubernetes cluster
+        host: kubernetes.default
+        # Resources for build pod
+        resources:
+          cpu:
+            # Number of cpu cores
+            micro: 1
+            low: 2
+            high: 6
+            turbo: 12
+            # upper bound for user custom cpu
+            max: 12
+          memory:
+            # Memory in GB
+            micro: 1
+            low: 2
+            high: 12
+            turbo: 16
+            # upper bound for user custom memory
+            max: 16
+        # Default build timeout for all builds in this cluster
+        buildTimeout: 90
+        # Default max build timeout
+        maxBuildTimeout: 120
+        # k8s node selectors for approprate pod scheduling
+        nodeSelectors: {}
+        preferredNodeSelectors: {}
+      # Launcher image to use
+      launchImage: screwdrivercd/launcher
+      # Launcher container tag to use
+      launchVersion: stable
+      # Circuit breaker config
+      fusebox:
+        breaker:
+          # in milliseconds
+          timeout: 10000
+      # requestretry configs
+      requestretry:
+        # in milliseconds
+        retryDelay: 3000
+        maxAttempts: 5
+
+httpd:
+  # Port to listen on
+  port: 9001
+  # Host to listen on (set to localhost to only accept connections from this machine)
+  host: 0.0.0.0
+  # Externally routable URI (usually your load balancer or CNAME)
+  # This requires to be a routable IP inside docker for executor, see
+  # https://github.com/screwdriver-cd/screwdriver/blob/095eaf03e053991443abcbde91c62cfe06a28cba/lib/server.js#L141
+  uri: http://IP_ADDRESS:PORT
+
+ecosystem:
+  # Externally routable URL for the User Interface
+  ui: http://IP_ADDRESS:PORT
+  # Externally routable URL for the API
+  api: http://IP_ADDRESS:PORT
+  # Externally routable URL for the Artifact Store
+  store: http://http://IP_ADDRESS:PORT
```

Comment:
typo ? 

Also missing  `custom-environment-variables.yaml` like what we have in `queue-worker` https://github.com/screwdriver-cd/queue-worker/blob/master/config/custom-environment-variables.yaml